src/lib/mina/account-update.ts

```diff
@@ -1008,9 +1008,24 @@ class AccountUpdate implements Types.AccountUpdate {
     return new AccountUpdateTree({ accountUpdate, id, children });
   }
 
+  /**
+   * @deprecated Use {@link AccountUpdate.default} instead.
+   */
   static defaultAccountUpdate(address: PublicKey, tokenId?: Field) {
+    return AccountUpdate.default(address, tokenId);
```

Comment:
it always annoyed me that `AccountUpdate.defaultAccountUpdate()` was so verbose, so I renamed it to `AccountUpdate.default()`